src/main/resources/reference.conf

```diff
@@ -138,6 +138,17 @@ cassandra-journal {
   # but you cannot combine BlogPosts and Authors, since they have the same tag identifier.  
   tags {
   }
+  
+  # Minimum time between publishing messages to DistributedPubSub to announce events for a specific tag have
+  # been written. These announcements cause any ongoing getEventsByTag to immediately re-poll, rather than
+  # wait. In order enable this feature, make the following settings:
+  #  
+  #    - enable clustering for your actor system
+  #    - cassandra-journal.pubsub-minimum-interval = 10ms            (send real-time announcements at most every 10ms)
```

Comment:
please adjust this to `1s`, I know it's just docs, but users will take it as a good default, and I'm scared about sending too much remote messages